tests/console/kdump_and_crash.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run 'crash' utility on a kernel memory dump
+# Maintainer: Michal Nowak <mnowak@suse.com>
+
+use base "console_yasttest";
+use strict;
+use testapi;
+use utils;
+use kdump_utils;
+
+sub run() {
+    my ($self) = @_;
+    select_console('root-console');
+
+    # preparation for crash test
+    prepare_for_kdump;
+    activate_kdump;
+
+    # restart to activate kdump
+    script_run 'reboot', 0;
+    $self->wait_boot;
+    reset_consoles;
+    select_console 'root-console';
+
+    # soft failure ref bsc#1022064 if kdump could not be enabled
+    return 1 unless kdump_is_active;
```

Comment:
There's already a bug for it: https://bugzilla.suse.com/show_bug.cgi?id=957053. And when I run tests, often it fails because kdump doesn't work properly. Should we always let the test fail?